lib/templates/schema.json

```diff
@@ -84,19 +85,36 @@
 					"title": "Library Name",
 					"description": "This is the name of the concatenated file.",
 					"placeholder": "Example: app"
+				},
```

Comment:
I added two new fields below because it will be easier to manage instead of reusing the other concat fields.

Reusing would probably be cleaner but it involves a massive code change so I do not want to go there for now.